portal-api/src/v1/api.js

```diff
@@ -269,6 +269,26 @@ const getUtilisationReports = async (bankId) => {
   }
 };
 
+const getBankById = async (bankId) => {
+  try {
+    if (!isValidBankId(bankId)) {
+      console.error('Get bank failed with the following bank ID %s', bankId);
+      return false;
+    }
```

Comment:
Seeing as this doesn't throw an error, couldn't it be moved to before the try-catch block?